### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by kristoferlund**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/enhancement-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: Enhancement issue template
+about: 'Use this template when suggesting new handbook content '
```
Comment (majg0) [inline .github/ISSUE_TEMPLATE/enhancement-issue-template.md]: Please trim end of spaces

File: .github/ISSUE_TEMPLATE/enhancement-issue-template.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Issue type: IDEA / ENHANCEMENT REQUEST
+
+## Driver
+
+**Delete the following paragraphs and replace with your driver description**
+
+What is the motive for this issue, what's happening? Describe the current situation, the effect of the situation on the organization and what’s needed. What is the the impact of attending to the need?
+
+*The handbook navigation doesn't look good my iPhone X5. That plain black-n-white stying makes us look lame as a political party. We need a manu navigation that feels more… 3D and futuristic. That way we could attract a lot more young voters.*
```
Comment (majg0) [inline .github/ISSUE_TEMPLATE/enhancement-issue-template.md]: "manu"

File: .github/ISSUE_TEMPLATE/new-content-issue-template.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+name: New content issue template
+about: 'Use this template when suggesting new handbook content '
```
Comment (majg0) [inline .github/ISSUE_TEMPLATE/new-content-issue-template.md]: Please trim end of spaces

File: .github/ISSUE_TEMPLATE/new-content-issue-template.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+title: ''
+labels: content
+assignees: ''
+
+---
+
+Issue type: NEW CONTENT
+Parent category: **Where should this content be placed?**
```
Comment (majg0) [inline .github/ISSUE_TEMPLATE/new-content-issue-template.md]: What is this? I think it needs elaboration, e.g. a list of valid parents or a link to documentation.

**Review by majg0 — CHANGES_REQUESTED** [review]
Looks good overall! :tada:

File: .github/ISSUE_TEMPLATE/new-content-issue-template.md (hunk 4 above)
Comment (kristoferlund) [reply .github/ISSUE_TEMPLATE/new-content-issue-template.md]: New text:

`Parent category: /category/subcategory (where in the table of contents should this content be placed)`

Not perfect. Let's clarify this further on.

Comment (majg0) [reply .github/ISSUE_TEMPLATE/new-content-issue-template.md]: Much better though! I'm satisfied with this for now.

**Comment by majg0** [comment]
Thanks @kristoferlund ! :tada:

**Pull request merged**